.github/workflows/ota-update.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    tags:
+      # matches "v1.2.1.ota-prod.8"
+      - 'v[0-9]+.[0-9]+.[0-9]+ota-prod.[0-9]+'
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
+
+      - name: 🏗 Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: yarn
+
+      - name: 🏗 Setup EAS
+        uses: expo/expo-github-action@v8.2.1
+        with:
+          eas-version: latest
```

Comment:
(optional) I'm not sure how it works with these `eas` versions, but in general, for production, it is better to pin a specific version, so you don't run into issues that something updates on you under the hood without you knowing it and breaks something.